R/boxr_read.R

```diff
@@ -12,18 +12,31 @@
 #' provided:
 #' 
 #' \describe{
-#'   \item{`box_read_csv()`}{parse a remote CSV file into a `data.frame` using [read.csv()]}
-#'   \item{`box_read_tsv()`}{parse a remote TSV file into a `data.frame` using [read.delim()]}
+#'   \item{`box_read_csv()`}{parse a remote CSV file into a `data.frame` using [data.table::fread()]
+#'     if available and [read.csv()] if not}
+#'   \item{`box_read_tsv()`}{parse a remote TSV file into a `data.frame` using [data.table::fread()]
+#'     if available and [read.delim()] if not }
 #'   \item{`box_read_json()`}{parse a remote JSON file into a `list` using [jsonlite::fromJSON()]}
 #'   \item{`box_read_excel()`}{parse a remote Microsoft Excel file into a `data.frame` using [readxl::read_excel()]}
 #' }
 #' 
+#' @section rio::import() and JSON files:
```

Comment:
A couple things for this block:

- could you enclose function names in `[]`, e.g. [rio::import()]
- could you enclose `data.frame` in ``